generators/readme/src/index.ts

```diff
@@ -56,6 +93,74 @@ const parseMarkdownFragmentToAST = (filePath: string) => {
 
   return fragmentAST.children;
 };
+const parsePropTypesToMarkdown = (componentPath: string) => {
+  const result = shelljs.exec(`react-docgen ${componentPath}`, {
+    silent: true,
+  });
+  const reactAPI: ReactAPI = JSON.parse(result.stdout);
+  const tableHeaders = tableRow([
+    tableCell('Props'),
+    tableCell('Type'),
+    tableCell('Required'),
+    tableCell('Values'),
+    tableCell('Default'),
+    tableCell('Description'),
+  ]);
+
+  const tableBody = Object.entries(reactAPI.props).map(
+    ([propName, propInfo]) => {
+      let propType: string;
+      let propDefaultValue: string = '';
+      switch (propInfo.type.name) {
+        case 'arrayOf':
+          const arrayValue = propInfo.type.value as {
+            name: string;
+            value?: unknown;
+          };
+          switch (arrayValue.name) {
+            case 'shape':
+              const arrayShapeValue = arrayValue.value as {
+                [name: string]: { name: string };
+              };
+              propType = Object.entries(arrayShapeValue)
+                .map(
+                  ([arrayShapePropName, arrayShapePropValue]) =>
+                    `{ ${arrayShapePropName}: ${arrayShapePropValue.name} }[]`
+                )
+                .join('\n');
+              break;
+            default:
+              propType = `${arrayValue.name}[]`;
+              break;
+          }
+          break;
+        case 'enum':
+          propType = propInfo.type.name;
+          propDefaultValue = (propInfo.type.value as { value: string }[])
+            .map((enumValue) => enumValue.value)
+            .join(', ');
+          break;
+        default:
+          propType = propInfo.type.value
+            ? String(propInfo.type.value)
+            : propInfo.type.name;
```

Comment:
shouldn't we update `propDefaultValue` here as well? Most of the times the props of type `boolean` has some sort of default value